projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -11,21 +11,21 @@
 <ng-template #inlineEditor let-cell="cell">
     <ng-container *ngIf="column.dataType === 'string'">
         <igx-input-group displayDensity="compact">
-            <input igxInput [value]="editValue" (input)="editValue = $event.target.value" [igxFocus]="true" />
+            <input igxInput [value]="editValue" (input)="editValue = $event.target.value" [igxFocus]="focused" />
         </igx-input-group>
     </ng-container>
     <ng-container *ngIf="column.dataType === 'number'">
         <igx-input-group displayDensity="compact">
-            <input igxInput [value]="editValue" (input)="editValue = $event.target.value" [igxFocus]="true" type="number">
+            <input igxInput [value]="editValue" (input)="editValue = $event.target.value" [igxFocus]="focused" type="number">
```

Comment:
Maybe bind to `active` instead?

Reply:
we need to move focus to the edit template only initially, the template should be blurred if we perform scroll, but the cell will remain active, so if we bind the igxFocus to active property .. this will retrigger focus over the template again and again on each change detection  which broke the scrolling experience in the Igx-HierarchicalGrid.